pkg/matcher/README.md

```diff
@@ -21,19 +27,30 @@ Short Syntax
 
  Long Syntax
 
-     <line>      ::= [ <not> ] <format> <separator> <expr>
+     [ <not> ] <format> <separator> <expr>
      <format>    ::= [ 'string' | 'glob' | 'regexp' | 'simple_patterns' ]
      <not>       ::= '!'
                        negative expression
      <separator> ::= ':'
      <expr>      ::= any string
 ```
 
+**Tip**: In short syntax, you can enable the glob format by starting the string with a `*`, while in the long syntax you need to define it more explicitly. The following examples are identical. `simple_patterns` can be used **only** with the long syntax.
+
+Short Syntax: `'* * '`
+Long Syntax: `'glob':'*'`
```

Comment:
i think we dont need this tip, all specific examples should be under appropriate matchers sections

Reply:
I want this example because it makes it explicit that the 2 formats are identical. it took me some time to understand what short and long format is, this example makes it dead obvious, so I would like to keep it.

Reply:
you added the table in the beginning of readme, that info in there

Reply:
Still, I prefer to over-communicate things that people may have already understand, that miss something that a few people will scratch their heads about. I feel quite strongly about this one tbh :P @joelhans wanna chime in ? 
In general my approach to documentation is "assume that you are talking to a small child, or a even a dog".